src/lib/Components/Consignments/Submission/geminiUploadToS3.ts

```diff
@@ -96,7 +95,9 @@ export const uploadFileToS3 = async (file: string, req: GeminiCredsInput, res: G
     const geminiKey = asset.policy_document.conditions.gemini_key
     const bucket = asset.policy_document.conditions.bucket
     const uploadURL = `https://${bucket}.s3.amazonaws.com`
-    const filename = file.split(/\//g).pop()
+
+    // FIXME: This is being unused in 103; does this uploader work?
+    // const filename = file.split(/\//g).pop()
```

Comment:
This is weird, check out https://github.com/artsy/emission/pull/877/files#diff-1e4221a2b438a2ae235d95aff0c14c71L103 (a few lines down from here)

```javascript
key: geminiKey + "/${filename}",
```
Is the uploader even working? Should have backticks for interpolation

Reply:
Yeah, it is weird, but that's actually what it expects ( this is the web version of this code: https://github.com/artsy/gemup/blob/master/gemup.js#L44 )